### PR TITLE
[#0] Adds a *destructive* command in our cjl script to reset the database

### PR DESCRIPTION
**Pull request by lancelafontaine**
[#0]

- [x] Adds a *destructive* command in our `cjl` script to reset the database on the local host

### CODE REVIEW

**Review by Vynny — APPROVED** [review]
LGTM

File: cjl [hunk 1]
```diff
@@ -21,6 +21,18 @@ if [ "$1" == "clean" ]; then
   exit
 fi
 
+if [ "$1" == "reset-db" ]; then
+  echo "WARNING: This command is destructive. It removes all database data on your local machine."
+  read -r -p "Are you SURE you want to continue? [y/N]" clean_confirmation
```
Comment (arekmano) [inline cjl:26]: wouldn't this make it harder to automate?

Comment (arekmano) [reply cjl:26]: i.e. for deployments?

Comment (dodoels) [reply cjl:26]: probably `echo Y`?

Comment (lancelafontaine) [reply cjl:26]: Would make sense to have a `-y` or `--noconfirm` flag to skip this check entirely.

**Pull request merged**